_monitoring-plugins/pa/index.md

```diff
@@ -83,122 +119,185 @@ Then remount the file system:
-
-   ```
-   # ======================== OpenSearch performance analyzer plugin config =========================
-
-   # NOTE: this is an example for Linux. Please modify the config accordingly if you are using it under other OS.
-
-   # WebService bind host; default to all interfaces
-   webservice-bind-host = 0.0.0.0
-
-   # Metrics data location
-   metrics-location = /dev/shm/performanceanalyzer/
-
-   # Metrics deletion interval (minutes) for metrics data.
-   # Interval should be between 1 to 60.
-   metrics-deletion-interval = 1
-
-   # If set to true, the system cleans up the files behind it. So at any point, we should expect only 2
-   # metrics-db-file-prefix-path files. If set to false, no files are cleaned up. This can be useful, if you are archiving
-   # the files and wouldn't like for them to be cleaned up.
-   cleanup-metrics-db-files = true
-
-   # WebService exposed by App's port
-   webservice-listener-port = 9600
-
-   # Metric DB File Prefix Path location
-   metrics-db-file-prefix-path = /tmp/metricsdb_
-
-   https-enabled = false
-
-   #Setup the correct path for certificates
-   certificate-file-path = specify_path
-
-   private-key-file-path = specify_path
-
-   # Plugin Stats Metadata file name, expected to be in the same location
-   plugin-stats-metadata = plugin-stats-metadata
-
-   # Agent Stats Metadata file name, expected to be in the same location
-   agent-stats-metadata = agent-stats-metadata
-   ```
-
-1. Make the CLI executable:
-
-   ```bash
-   sudo chmod +x ./bin/performance-analyzer-agent-cli
-   ```
-
-1. Launch the agent CLI:
-
-   ```bash
-   OPENSEARCH_HOME="$PWD" OPENSEARCH_PATH_CONF="$PWD/config" ./bin/performance-analyzer-agent-cli
-   ```
-
-1. In a separate window, enable the Performance Analyzer plugin:
+```bash
+curl -XPOST http://localhost:9200/_plugins/_performanceanalyzer/rca/cluster/config -H 'Content-Type: application/json' -d '{"enabled": true}'
+```
 
-   ```bash
-   curl -XPOST localhost:9200/_plugins/_performanceanalyzer/cluster/config -H 'Content-Type: application/json' -d '{"enabled": true}'
-   ```
+If you run into the `curl: (52) Empty reply from server` response, run the command below to enable RCA.
 
-   If you receive the `curl: (52) Empty reply from server` error, you are likely protecting your cluster with the security plugin and you need to provide credentials. Modify the following command to use your username and password:
+```bash
+curl -XPOST https://localhost:9200/_plugins/_performanceanalyzer/rca/cluster/config -H 'Content-Type: application/json' -d '{"enabled": true}' -u 'admin:admin' -k
+```
 
-   ```bash
-   curl -XPOST https://localhost:9200/_plugins/_performanceanalyzer/cluster/config -H 'Content-Type: application/json' -d '{"enabled": true}' -u 'admin:admin' -k
-   ```
+### Example API query and response
+
+Request all available RCAs.
+
+````bash
+GET localhost:9600/_plugins/_performanceanalyzer/rca
+````
+
+Request a specific RCA.
```

Comment:
Make a complete sentence. "To request a specific RCA, run the following command:".